Detectors/CTF/test/test_ctf_io_trd.cxx

```diff
@@ -52,7 +52,7 @@ BOOST_AUTO_TEST_CASE(CTFTest)
 
       for (int i = nTrk; i--;) {
         tracklets.emplace_back(formatTrk, hcid, gRandom->Integer(0x1 << 4), gRandom->Integer(0x1 << 2),
-                               gRandom->Integer(0x1 << 11), gRandom->Integer(0x1 << 8), gRandom->Integer(0x1 << 24));
+                               gRandom->Integer(0x1 << 11) ^ 0x80, gRandom->Integer(0x1 << 8) ^ 0x80, gRandom->Integer(0x1 << 24));
```

Comment:
here too

Reply:
Its done in the electronics, so the post and slope stored in the tracklet64 class has the bit flipped.

Reply:
ok, so this is just to emulate the weirdness of the firmware...

Reply:
oui

Reply:
In principle, we would have to keep the data fields as not bitflipped in Tracklet64, and do the conversion when decoding the raw data to tracklets (and digits).
Some background on the weirdness: the flipped bits are necessary to avoid misinterpretation of a tracklet word as a tracklet data end marker.

Reply:
> Some background on the weirdness: the flipped bits are necessary to avoid misinterpretation of a tracklet word as a tracklet data end marker.

Does this mean that the flipped bit is always there?

Reply:
the bit flipped data is stored inside the tracklet64 class, hence the conversion in the get methods. 
This leaves the current ctf alone.

Reply:
Would it make sense to remove the change in this line, since it is not relevant (the actual fix is in the lines below)? Or do you think it makes the test more readable? Then we can leave it of course. 
That would also kick the CI again and then we can hopefully get this merged soon

Reply:
Sure it does seem rather random, to flip a bit, to someone not familiar reading the code. Or even us in 3 years time.